aws/app/lambda.tf

```diff
@@ -458,7 +458,7 @@ resource "aws_lambda_layer_version" "audit_logs_lib" {
   filename            = "/tmp/audit_logs_lib.zip"
   layer_name          = "audit_logs_node_packages"
   source_code_hash    = data.archive_file.audit_logs_lib.output_base64sha256
-  compatible_runtimes = ["nodejs12.x", "nodejs14.x"]
+  compatible_runtimes = ["nodejs18.x"]
 }
```

Comment:
Would you know if or staging and production environments have pretty much the same setup/versions?

Reply:
Weird... Some docs say 18 is available but some other AWS docs say 16...  switched to 16 for now.